src/frequenz/sdk/actor/_actor.py

```diff
@@ -54,6 +54,19 @@ def start(self) -> None:
     async def _run(self) -> None:
         """Run this actor's logic."""
 
+    async def _delay_restart(self, iteration: int) -> None:
+        """Delay the restart of this actor's n'th iteration.
+
+        Args:
+            iteration: The current iteration of the restart.
+        """
+        # NB: I think it makes sense (in the future) to think about deminishing returns
+        # the longer the actor has been running.
+        # Not just for the restart-delay but actually for the n_restarts counter as well.
+        delay = 2**iteration
```

Comment:
we have a generic retry time calculation thing here: https://github.com/frequenz-floss/frequenz-sdk-python/blob/356200443f07acc67b17ba61b0718e7db9643852/src/frequenz/sdk/microgrid/client/_retry.py

we could have a basic LinearBackoff once every 2 seconds as default, but add a method that allows actors to override the strategy.